requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
Would like to move this to test-requirements.txt, but it appears to be used in non-test modules ([example](https://github.com/dimagi/commcare-hq/blob/3635a60/corehq/apps/app_manager/models.py#L5)).